tests/unittest/_torch/auto_deploy/unit/singlegpu/transformations/library/test_gated_delta_rule_cache.py

```
@@ -93,33 +96,31 @@ def __init__(
         self.v_proj = nn.Linear(hidden_size, num_heads * value_dim, bias=False)
-        self.g_proj = nn.Linear(hidden_size, num_heads, bias=False)
-        self.beta_proj = nn.Linear(hidden_size, num_heads, bias=False)
+        self.a_proj = nn.Linear(hidden_size, num_heads, bias=False)
+        self.b_proj = nn.Linear(hidden_size, num_heads, bias=False)
+        self.A_log = nn.Parameter(torch.zeros(num_heads))
+        self.dt_bias = nn.Parameter(torch.zeros(num_heads))
         self.o_proj = nn.Linear(num_heads * value_dim, hidden_size, bias=False)
 
     @torch.no_grad()
     def forward(
         self, input_ids: torch.Tensor, position_ids: Optional[torch.Tensor] = None
```

Ruff (0.15.2)

[warning] Unused method argument: `position_ids`

(ARG002)